modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -1,22 +1,25 @@
-= Amazon EC2 Connector
+= Amazon EC2 Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: anypoint studio, connector, ec2, amazon ec2, user guide
+:imagesdir: ../../assets/images/
 
-_Select_
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-The Anypoint Connector for Amazon EC2 provides connectivity to the Amazon Elastic Compute Cloud (Amazon EC2) API,
+Anypoint Connector for Amazon EC2 provides connectivity to the Amazon Elastic Compute Cloud (Amazon EC2) API,
 enabling you to interface with Amazon EC2 to launch as many or as few virtual servers as you need, configure security and networking, and manage storage.
 
+Release Notes: xref:release-notes::connector/amazon-ec2-connector-release-notes-mule-4.adoc[Amazon EC2 Connector Release Notes] +
+Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-amazon-ec2-connector/[Amazon EC2 Connector - Mule 4]
+
 [[prerequisites]]
 == Prerequisites
 
 To use the Amazon EC2 Connector, you must have the following:
 
 * Access to Amazon Web Services - EC2.
-* To access AWS with the connector, you need AWS Identity and Access Management (IAM) user account credentials.
-* Anypoint Studio version 7.0 (or higher) or Anypoint Design Center.
+* AWS Identity and Access Management (IAM) user account credentials.
+* Anypoint Studio version 7 or Anypoint Design Center.
```

Comment:
no periods: neither is a sentence
GLOBAL when none of the list items is a sentence.